homeassistant/components/synology/camera.py

```diff
@@ -51,6 +51,24 @@
 
     cameras = surveillance.get_all_cameras()
 
+    
+    def handle_enable_home_mode(call):
+    """Register a service to be able to enable home mode."""
+        surveillance.set_home_mode(True)
+
+    hass.services.async_register(DOMAIN,
```

Comment:
These should not be registered under the camera domain but under synology domain